CONTRIBUTING.md

```diff
@@ -1,22 +1,49 @@
+##### TC39 meetings and champions
+If you have a new proposal you want to get into the language, you first need a TC39 "champion": a member of the committee who will make the case for the proposal at physical TC39 meetings and help it move through the process. If you are a TC39 member, you can be a champion; otherwise, find a TC39 member to work with for this (e.g., through es-discuss or #tc39). Proposals may have multiple champions (a "champion group").
 
-### Signing the CLA
-If you wish to submit a proposal and are not a representative of a TC39 member, here are the steps you need to take:
+TC39 meets six times a year, mostly in the United States, to discuss proposals. It is possible for members to join meetings remotely. At meetings, we discuss ways to resolve issues and feature requests. We spend most of the time considering proposals and advancing them through the stage process. Meetings follow an agenda which is developed in the [agendas GitHub repository](https://github.com/tc39/agendas/). After the meeting, notes are published in the [notes GitHub repository](https://github.com/tc39/tc39-notes/). To advance your proposal towards inclusion in the final specification, ensure that it is included on the agenda for an upcoming meeting and propose advancement at that time.
 
-  1. Read the [TC39 process document](https://tc39.github.io/process-document/).
-  2. [Register as a TC39 contributor](https://ecma-international.org/memento/register_TC39_Royalty_Free_Task_Group.php) (it is not necessary to submit the contribution as attachment to the form)
+##### Helping with existing proposals
+TC39 is currently considering adding several new features to the language. These proposals are linked from [the proposals repository](https://github.com/tc39/proposals). There are many ways to help with existing proposals:
+  * File issues in the individual proposal repository to provide constructive criticism and feedback.
+  * Make PRs against proposals, e.g., to clarify explanations of the motivation and use cases in README.md, or to fix issues in the proposal's specification text.
+  * Talk about what you think of the proposal, including sharing thoughts with the champion.
+  * Blog, tweet, give talks, etc about proposals to get more awareness and programmer feedback about them.
+  * Write [test262](https://github.com/tc39/test262) tests against the proposal, which are used to verify implementations. (If the proposal is at Stage 3, the tests can land; if earlier, they can be maintained in a PR.)
+  * Implement or prototype the proposal in an ECMAScript implementation, parser, polyfill or development tool such as es6draft, JSC, Duktape, V8, SpiderMonkey, ChakraCore, Babel, TypeScript, Acorn, Flow, Esprima, Closure, Uglify, core-js, es-shims, etc., possibly behind a flag or in a separate module, and report feedback. Note that proposals before Stage 3 are highly unstable, and all proposals before Stage 4 are may be modified or dropped.
```

Comment:
This list seems too large (but also incomplete)... If we want to enumerate implementations, we could do it in a separate repo with nice categorized lists (script engines, parsers, compilers, polyfills...)